app/controllers/holds_controller.rb

```diff
@@ -149,7 +135,7 @@ def handle_not_needed_after_request
     end
 
     def process_flash(type, translation)
-      flash[type] = "#{flash[type]} #{t "myaccount.hold.#{translation}", bib_summary: @hold_to_act_on.bib_summary}"
```

Comment:
this was removed just to get rails to stop complaining, it'll be fixed in a PR soon